lib/bindConnectors/bindDynamicOutput.js

```diff
@@ -0,0 +1,31 @@
+var _                = require('lodash');
+var when             = require('when');
+var GenerateSchema = require('generate-schema');
+
+module.exports = function (message, options) {
+
+    return function (event) {
+
+        return when.promise(function (resolve, reject) {
+
+            when(message.dynamicOutput(event.body.input))
```

Comment:
Ok. @adriantoine, keep it as it is, don't wrap it in `input`. I'll change it to `event.body`.